packaging/os/pulp_repo.py

```diff
@@ -0,0 +1,568 @@
+    description:
+      - Proxy url setting for the pulp repository importer. This is in the
+        format scheme://host.
+    required: false
+    default: null
+  proxy_port:
+    description:
+      - Proxy port setting for the pulp repository importer.
+    required: false
+    default: null
+  importer_ssl_ca_cert:
+    description:
+      - CA certificate string used to validate the feed source SSL certificate.
+    required: false
+    default: null
+  importer_ssl_client_cert:
+    description:
+      - Certificate used as the client certificate when synchronizing the
+        repository. This is used to communicate authentication information to
+        the feed source. The value to this option must be the full path to the
+        certificate. The specified file may be the certificate itself or a
+        single file containing both the certificate and private key.
+    required: false
+    default: null
+  importer_ssl_client_key:
+    description:
+      - Private key to the certificate specified in I(importer_ssl_client_cert),
+        assuming it is not included in the certificate file itself.
+    required: false
+    default: null
+  relative_url:
+    description:
+      - Relative URL for the local repository.
+    required: true
+    default: null
+  add_export_distributor:
+    description:
+      - Whether or not to add the export distributor to new C(rpm) repositories.
+    required: false
+    default: false
+  serve_http:
+    description:
+      - Make the repo available over HTTP.
+    required: false
+    default: false
+  serve_https:
+    description:
+      - Make the repo available over HTTPS.
+    required: false
+    default: true
+  publish_distributor:
+    description:
+      - Distributor to use when state is C(publish). The default is to
+        publish all distributors.
+    required: false
+  url_username:
+    description:
+      - The username for use in HTTP basic authentication to the pulp API.
+    required: false
+  url_password:
+    description:
+        - The password for use in HTTP basic authentication to the pulp API.
+          If the C(url_username) parameter is not specified, the C(url_password)
+          parameter will not be used.
+    required: false
+  force_basic_auth:
+    description:
+      - httplib2, the library used by the M(uri) module only sends authentication
+        information when a webservice responds to an initial request with a 401
+        status. Since some basic auth services do not properly send a 401,
+        logins will fail. This option forces the sending of the Basic
+        authentication header upon initial request.
+    required: false
+    choices: [ "yes", "no" ]
+    default: "no"
+  validate_certs:
+    description:
+      - If C(no), SSL certificates will not be validated. This should only be
+        used on personally controlled sites using self-signed certificates.
+    required: false
+    default: 'yes'
+    choices: ['yes', 'no']
```

Comment:
Please add spaces around the parentheses and use double quotes in order to have it the same across the all DOCUMENTATION block:

```
choices: [ "yes", "no" ]
```